source/documentation/tools/create-a-derived-table/index.md

```diff
@@ -265,7 +256,7 @@ If you're working on a data model as part of a team you can take inspiration fro
 
```

Comment:
On L255 (_it wont let me comment there for some reason_) spelling mistake "_changrs_"